daemon/src/db.rs

```diff
@@ -357,6 +357,27 @@ where
 ///     Cases: Collaborative settlement, CET, Refund
 /// 2. Event that fails the CFD early was recorded, meaning it becomes irrelevant for processing
 ///     Cases: Setup failed, Taker's take order rejected
+pub fn load_all_open_cfds<C>(
```

Comment:
❓ Is it intentional that the docstring above was assigned to this new function?

Reply:
I think the answer is yes. I suppose the idea is that this is the public API now. But actually we still rely on `load_open_cfd_ids` in `daemon::oracle`.

Reply:
Yeah, the fact that we rely on that in `oracle` is a bit of an inconvenience!